charts/sysdig-deploy/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 name: sysdig-deploy
 description: A chart with various Sysdig components for Kubernetes
 type: application
-version: 1.31.1
+version: 1.31.2
```

```suggestion
version: 1.32.0
```
We are releasing a new feature, so I'd bump at least the `minor` ;)